pkg/apis/pmemcsi/v1alpha1/deployment_types.go

```diff
@@ -0,0 +1,427 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package v1alpha1
+
+import (
+	"reflect"
+
+	corev1 "k8s.io/api/core/v1"
+	apiextensions "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// DeviceMode type decleration for allowed driver device managers
+type DeviceMode string
+
+const (
+	// DeviceModeLVM represents 'lvm' device manager
+	DeviceModeLVM DeviceMode = "lvm"
+	// DeviceModeDirect represents 'direct' device manager
+	DeviceModeDirect DeviceMode = "direct"
+)
+
+// DeploymentSpec defines the desired state of Deployment
+type DeploymentSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
```

Comment:
Can you replace this with instructions for how to do that in PMEM-CSI?

Reply:
Adde the `make` instruction.